conans/client/cmake.py

```diff
@@ -330,6 +340,11 @@ def _build_new(self, args=None, build_dir=None, target=None):
         command = "cmake --build %s" % arg_list
         self._conanfile.run(command)
 
+    def package(self):
```

Comment:
Yes, I know, but, It's not mandatory that the CMake install installs libs in the system right? Actually here is copying the things to the package folder.
if cmake.configure performs the project configure, if cmake.build performs the build, if cmake.test runs the cmake tests... why not cmake.install? Is not executing an install? (the destination is not relevant)